.travis.yml

```diff
@@ -72,7 +77,7 @@ deploy:
       tags: true
       all_branches: true
       # if it's a PROD tag (something like 1.0.0), then publish extension to market.
-      condition: "$TRAVIS_TAG =~ $ISPRODTAG"
+      condition: "$IS_PROD = true"

```

Comment:
Is the value of $IS_PROD calculated or configured in env variable?

Reply:
Fxied. Nice catch.